subprojects/core/src/main/java/org/gradle/api/tasks/AbstractExecTask.java

```diff
@@ -362,4 +362,21 @@ void setExecAction(ExecAction execAction) {
     public ExecResult getExecResult() {
         return execResult;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Internal
+    @Override
+    public boolean isSpawn() {
+        return execAction.isSpawn();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void setSpawn(boolean spawn) {
+        execAction.setSpawn(spawn);
+    }
```

Comment:
Instead of using the java bean style getter/setter pattern this should probably expose a single method:
```java
public Property<Boolean> getSpawn()
```
However, I'm not a member of the gradle team so I'd wait on their feedback before doing that.

Reply:
CC: @big-guy